internal/proxyservice/proxy_service.go

```diff
@@ -123,9 +123,17 @@ func (s *proxyServices) createHTTPService(
 		curConnector := curPlugin.NewConnector(connResources)
 		credsRetriever := s.credsRetriever(subCfg.Credentials)
 
+		// Get the http traffic patterns to match from the connector config.
+		httpCfg, err := v2.NewHTTPConfig(subCfg.ConnectorConfig)
+		if err != nil {
+			return nil, err
+		}
+
 		subservices = append(subservices, httpproxy.Subservice{
-			Connector:           curConnector,
-			RetrieveCredentials: credsRetriever,
+			ConnectorID:              subCfg.Connector, // TODO: Rename connectorID
```

Comment:
As far as I can tell ConnectorID is used in a single place for logging. Wondering if it's overkill to have it as a field on the subservice.

Reply:
Yeah there's two perspectives here.  One is the narrower one which is, "Eh, let's just remove it _and_ the log statement and keep it simple."  The larger one, in my mind, which I was hinting at with this:

> // TODO: Looks like ConnectorID and the function it represents shouldn't be separated

is that this "little" problem with the log is actually a smell that our design is subtly wrong, and the the concept of a Connector, as we think about it, isn't just a raw function, but at minimum a function _and_ and an id/name.  This hasn't bit us except here, and so practically we can probably ignore it, at least for now, but I do believe it's a small design error.

Reply:
I agree. I suppose we can reevaluate this as more information comes to light.